app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentSelfAuthAdds.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString + " update") {
+
```

Comment:
Yeah.. I know.. but as it is a general service and would be used by many more services in future, we cannot add more. Maybe in future after loads of user research sessions..